static/app/views/performance/utils.tsx

```diff
@@ -71,8 +71,10 @@ export function platformToPerformanceType(
   }
 
   if (
-    selectedProjects.every(project =>
-      BACKEND_PLATFORMS.includes(project.platform as string)
+    selectedProjects.every(
+      project =>
+        BACKEND_PLATFORMS.includes(project.platform as string) &&
+        project.platform !== 'native'
```

Comment:
can we add a comment here saying the reason why we would like "native" to not be on the list?